### PR TITLE
Allow pre-releases in propose-update triggered by comment

### PR DESCRIPTION
**Pull request by lbarcziova**
Should fix: https://sentry.io/organizations/red-hat-0p/issues/1566953230/?project=1767823&query=is%3Aunresolved

### CODE REVIEW

File: packit_service/worker/handlers/github_handlers.py [hunk 1]
```diff
@@ -672,29 +671,13 @@ def __init__(self, config: ServiceConfig, event: IssueCommentEvent):
...
         self.package_config.upstream_project_url = event.project_url
 
-    def get_tag_name(self) -> Optional[str]:
-        """
-        Get the tag name of the latest upstream release.
-
-        :return: tag name
-        """
-        try:
-            tag_name = self.project.get_latest_release().tag_name
```
Comment (lachmanfrantisek) [inline packit_service/worker/handlers/github_handlers.py]: What about  adding `allow_prereleases=False` argument to `get_latest_release` in ogr?

Comment (lbarcziova) [reply packit_service/worker/handlers/github_handlers.py]: I am not sure I get it, do you mean it in the way:
`allow_prereleases=False` -> call get_latest_release() in PyGithub
`allow_prereleases=True` -> workflow used in this PR?

Comment (lbarcziova) [reply packit_service/worker/handlers/github_handlers.py]: @lachmanfrantisek Ok, I will add it to the ogr, should I merge this anyway or wait for it?

Comment (lachmanfrantisek) [reply packit_service/worker/handlers/github_handlers.py]: No need to wait, merge it... thanks

**Comment by lsm5** [comment]
@lachmanfrantisek @lbarcziova please help me understand, are pre-releases in downstream currently supported by comment triggers but not automatically ?

**Comment by lachmanfrantisek** [comment]
@lsm5 Hi! It's been a while since this was merged and a lot has changed...;)

I am checking the current state:
* We react to ["release" GitHub event](https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads?actionType=released#release):
> A release was published, or a pre-release was changed to a release.
* And with comments, [this](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-the-latest-release) is used internally:
> View the latest published full release for the repository.
> The latest release is the most recent non-prerelease, non-draft release, sorted by the created_at attribute. The created_at attribute is the date of the commit used for the release, and not the date when the release was drafted or published.


You are interested in the releases marked on GitHub side as "pre-release", right?

**Pull request merged**